gp_services.R

```diff
@@ -13,23 +13,23 @@ library(RJSONIO)
 services <- fromJSON("https://ual.geoplatform.gov/api/communities/4eebc494059beab9fda54cb078927ddc/items?type=regp:Service&size=200")
 
 #Create an empty vector to hold all of the service IDs
-service_ids <- c()
-service_label <- c() 
+id <- c()
+title <- c() 
 
 #Loop through each service entry in the JSON file (now an R list)
 for(i in 1:length(services[[1]])){
-  service_ids[i] <- services[[1]][[i]][['id']] 
-  service_label[i] <- services[[1]][[i]][['label']] 
+  id[i] <- services[[1]][[i]][['id']] 
+  title[i] <- services[[1]][[i]][['label']] 
 }
 
 #Create an empty vector to hold all of the service urls
-service_urls <- c()
+serviceUrl <- c()

```

Comment:
follow either snake case or camel case for variable names. (either `service_url` or `serviceUrl`) consistency in naming variables.